pages/publicfiles/PublicFilesHandler.inc.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/**
+ * @file pages/publicfiles/PublicFilesHandler.inc.php
+ *
+ * Copyright (c) 2013-2015 Simon Fraser University Library
+ * Copyright (c) 2003-2015 John Willinsky
+ * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+ *
+ * @class PublicFilesHandler
+ * @ingroup pages_publicfiles
+ *
+ * @brief Handle requests for public files functions.
+ */
+
+import('classes.handler.Handler');
+import('lib.pkp.classes.file.FileManager');
+
+class PublicFilesHandler extends Handler {
+	
+	/* @var fileManager FileManager object (see lib/pkp/classes/file/FileManager.inc.php) */
+	var $fileManager;
+	
+	//Constructor
+	function PublicFilesHandler() {
+		$this->fileManager = new FileManager();
+	}
+	
+	/**
+	 * Delete a file in a user's public files directory
+	 * @param $args array
+	 * @param $request PKPRequest
+	 * @return bool
+	 */
+	function delete($args, $request) {
```

Comment:
Thanks for the review! - 
Now the user has to have at least ROLE_ID_PROOFREADER as highest role to be able to delete a file in the directory [files_dir]/publicuploads/[journalId]